modules/sso.php

```diff
@@ -732,7 +851,27 @@ function allowed_redirect_hosts( $hosts ) {
 		return array_unique( $hosts );
 	}
 
-	function button( $args = array() ) {
+	/**
+	 * Builds the "Login to WordPress.com" button that is displayed on the login page as well as user profile page.
+	 * @param  array   $args       An array of arguments to add to the SSO URL.
+	 * @param  boolean $is_primary Should the button have the `button-primary` class?
+	 * @return string              Returns the HTML markup for the button.
+	 */
+	function build_sso_button( $args = array(), $is_primary = false ) {
+		$url = $this->build_sso_button_url( $args );
+		$classes = $is_primary
+			? 'jetpack-sso button button-primary'
+			: 'jetpack-sso button';
+
+		return sprintf( '<a href="%1$s" class="%2$s">%3$s</a>', esc_url( $url ), $classes, esc_html__( 'Log in with WordPress.com', 'jetpack' ) );
```

Comment:
Could use `wp_kses` restricting it to `array( 'a' => array( 'href' => true, 'class' => true ) )`